### ISSUE TITLE
Error when importing library obs_websocket then running on web environment

### ISSUE THREAD

**Issue by duchieupham** [faithoflifedev/obs_websocket#9]
I've just imported this line into my code: 

`import 'package:obs_websocket/obs_websocket.dart';`

Then the compiler warns me when I'm running on web environment:

```
.../.pub-cache/hosted/pub.dartlang.org/obs_websocket-2.2.12/lib/src/obs_websocket.dart:55:37:  
Error: The argument type 'WebSocket/*1*/' can't be assigned to the parameter type 'WebSocket/*2*/'.    
 - 'WebSocket/*1*/' is from 'package:universal_io/src/http/http.dart'
 ('/D:/flutter/.pub-cache/hosted/pub.dartlang.org/universal_io-2.0.4/lib/src/http/http.dart').
 - 'WebSocket/*2*/' is from 'dart:_http'.
        channel: IOWebSocketChannel(websocket), fallbackEvent: fallbackEvent);
                                    ^

```
Please help me this issue!

**Comment by faithoflifedev** [comment]
Sorry @duchieupham,  'web' is not in the list of supported platforms for the package.

**Comment by faithoflifedev** [comment]
Hi @duchieupham , I just published version 2.3.0 of the package that has web support.  I haven't had a chance to test the web functionality in detail though, so if you can give some feedback I'd appreciate it.  Thanks.

**Comment by faithoflifedev** [comment]
I've confirmed that web is working in version 2.3.1
